rules/microsoft-365/persistence_exchange_suspicious_mailbox_right_delegation.toml

```diff
@@ -0,0 +1,48 @@
+[metadata]
+creation_date = "2021/05/17"
+maturity = "production"
+updated_date = "2021/06/22"
+
+[rule]
+author = ["Elastic", "Austin Songer"]
+description = """
+Identifies the assignment of rights to accesss content from another mailbox. This is usually only assigned to a service
+account.
+"""
+false_positives = ["Unknown"]
```

Comment:
As of the Description, if someone uses this to assign this right to a service account, it would be a False positive, right?